tests/admin_test.py

```diff
@@ -36,32 +36,42 @@ def test_entry_to_dict(self):
             ('c', 'C', int),
         ]
 
-        self.assertEquals({'a': '1', 'b': ['x'], 'C': 1},
-                          admin._entry_2_dict({'a': ['1'],
-                                               'b': ['x'],
-                                               'c': ['1']}, schema))
+        self.assertEqual(
+            {'a': '1', 'b': ['x'], 'C': 1},
+            admin._entry_2_dict(
+                {'a': ['1'], 'b': ['x'], 'c': ['1']}, schema
+            )
+        )
 
-        self.assertEquals({'a': ['1'], 'b': ['x'], 'c': ['1']},
-                          admin._dict_2_entry({'a': '1',
-                                               'b': ['x'],
-                                               'C': 1}, schema))
+        self.assertEqual(
+            {'a': ['1'], 'b': ['x'], 'c': ['1']},
+            admin._dict_2_entry(
+                {'a': '1', 'b': ['x'], 'C': 1}, schema
+            )
+        )
 
     def test_group_by_opt(self):
         """Tests group by attribute option."""
         # Disable W0212: Test access protected members of admin module.
         # pylint: disable=W0212
-        self.assertEquals({'a': [('xxx', 'a', ['1']),
-                                 ('yyy', 'a', ['2'])],
-                           'b': [('xxx', 'b', ['3'])]},
-                          admin._group_entry_by_opt({'xxx;a': ['1'],
-                                                     'xxx;b': ['3'],
-                                                     'yyy;a': ['2']}))
+        self.assertEqual(
+            sorted(
```

Comment:
`admin._group_entry_by_opt(...`'s output order is not predictable, it returns results in different order in each run, which makes this test flaky. So, we have to sort the result to compare.